### PR TITLE
Update meta.yaml with more tests

### PR DESCRIPTION
**Pull request by gsheni**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,9 @@ source:
 
 build:
   number: 0
```
Comment (gsheni) [inline recipe/meta.yaml:12]:
```suggestion
  number: 1
```

**Comment by gsheni** [comment]
@conda-forge-admin, please rerender

**Review by thehomebrewnerd — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,8 +9,10 @@ source:
   sha256: f3aee2e7a1a701b792e008971dae207e80f24b2088f6846e6a834ac77ce67c85
 
 build:
-  number: 0
-
+  number: 1
+  entry_points:
+   - featuretools_primitives = nlp-primitives.nlp_primitives
```
Comment (gsheni) [inline recipe/meta.yaml:14]: Another idea (if above doesn't work)
```suggestion
   - featuretools_primitives = nlp-primitives:__main__
```

Comment (gsheni) [inline recipe/meta.yaml:14]:
```suggestion
   - featuretools_primitives = nlp-primitives.__main__
```

Comment (gsheni) [inline recipe/meta.yaml:14]:
```suggestion
   - featuretools_primitives = nlp-primitives
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -9,8 +9,10 @@ source:
   sha256: f3aee2e7a1a701b792e008971dae207e80f24b2088f6846e6a834ac77ce67c85
 
 build:
-  number: 0
-
+  number: 1
+  entry_points:
+   - featuretools_primitives = nlp-primitives
```
Comment (gsheni) [inline recipe/meta.yaml:14]:
```suggestion
   - featuretools_primitives = nlp-primitives:__init__
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -9,8 +9,10 @@ source:
   sha256: f3aee2e7a1a701b792e008971dae207e80f24b2088f6846e6a834ac77ce67c85
 
 build:
-  number: 0
-
+  number: 1
+  entry_points:
+   - featuretools_primitives = nlp-primitives:__init__
```
Comment (gsheni) [inline recipe/meta.yaml:14]:
```suggestion
   - featuretools_primitives = nlp-primitives.nlp_primitives
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -27,10 +26,12 @@ outputs:
         - featuretools >=1.5.0
         - nltk >=3.4.5
         - scikit-learn >=0.20.0
-        - woodwork >=0.8.1
+        - woodwork >=0.8.1,!= 0.20.0
```
Comment (gsheni) [inline recipe/meta.yaml:29]:
```suggestion
        - woodwork >=0.19.0, != 0.20.0
```

**Comment by gsheni** [comment]
@conda-forge-admin, please rerender

**Pull request merged**